#!/usr/bin/env python
PACKAGE = "perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("median_blur_size", int_t, 0,"the size of the median blur (must be odd!), zero to deactivate median blur", 3, 0, 19)

exit(gen.generate(PACKAGE, "perception", "PreprocessingMedianBlur"))
